Tests/AlicerceTests/DeepLinking/Route+TrieRouter_RouteTests.swift

```diff
@@ -130,6 +130,16 @@ class Route_TrieRouter_RouteTests: XCTestCase {
         XCTAssertRouteSucceeds(initial: [("/", testHandler)], route: "://")
     }
 
+    func testRoute_WithCaseInsensetiveScheme_shouldSucceed() {
```

Comment:
typo in "Insensetive", and "Should" should start with uppercase

also, perhaps:
`testRoute_WithCaseInsensitiveMatchingScheme_ShouldSucceed()`
`testRoute_WithCaseInsensitiveMatchingHost_ShouldSucceed()`

And maybe use some weird variations like `HttP` and `HoSt`, to make it clearer? (ie. not just CAPS vs caps)